frontend/test/resolution/testLoopIndexVars.cpp

```diff
@@ -148,13 +144,15 @@ static void testAmbiguous() {
   const ResolutionResultByPostorderID& rr = resolveModule(context, m->id());
   auto idx = rr.byAst(loop->index());
   assert(idx.type().isErroneousType());
-  assert(errors.size() == 1);
+  assert(guard.errors().size() == 1);
+  assert(guard.realizeErrors());
```

Comment:
`realizeErrors` returns `errors.size()`. Why not just `assert(guard.realizeErrors() == 1)`?